app/server/api/auth_api.py

```diff
@@ -86,7 +88,7 @@ def post(self):
         exact_match = False
 
         tier = None
-        sempoadmin_emails = current_app.config['SEMPOADMIN_EMAILS']
+        sempoadmin_emails = config.SEMPOADMIN_EMAILS
```

Comment:
Since I was adding a dependency in that file to config.py for the multi-chain stuff anyway, I wanted to make it internally consistent (I.e. I didn't want to have to import both `current_app` and `config` for the same thing).  
IMO we should do this everywhere to make it completely consistent